windows.md

```diff
@@ -193,8 +193,22 @@ With those compatibility things out of the way, you're ready to start the system
 
     1. Find `shell: '',` and replace it with `shell: 'C:\\Program Files\\Git\\bin\\bash.exe',`
     2. Find `env: {},` and replace it with `env: { TERM: 'cygwin' },`
+    3. Save the file and close and restart Hyper
 
-    Then save the file and close and restart Hyper.
+    To test that the Hyper setup was successful, you can copy and paste the following command into Hyper and hit return:
+    
+    ```bash
+    echo $SHELL; echo $TERM
+    ```
+
+    You should see the following output:
+   
+    ```bash
+    C:\Program Files\Git\bin\bash.exe
+    cygwin
+    ```
+    
+    If you see this output, then the Hyper setup was successful. If the output is different, then you need to go back to step 1 and double-check your changes
```

Comment:
going back to step 1 seems like a long way back 😬  (this is step 16 already) is it possible to do a smaller step back?

Reply:
Changed it, now it says go back to step 16
[d299d98](https://github.com/upleveled/system-setup/pull/32/commits/d299d98bc3aba9bd5b895910fb57503e73f8d8a1)